apps/contentful-apps/components/editors/lists/GenericListEditor/GenericListEditor.tsx

```diff
@@ -1,27 +1,14 @@
-import { useMemo, useRef, useState } from 'react'
-import { useDebounce } from 'react-use'
-import { CollectionProp, EntryProps, KeyValueMap } from 'contentful-management'
+import { useMemo, useState } from 'react'
 import dynamic from 'next/dynamic'
 import { EditorExtensionSDK } from '@contentful/app-sdk'
-import {
-  Box,
-  Button,
-  EntryCard,
-  FormControl,
-  Pagination,
-  Spinner,
-  Stack,
-  Text,
-  TextInput,
-} from '@contentful/f36-components'
+import { Box, Button, FormControl } from '@contentful/f36-components'
 import { PlusIcon } from '@contentful/f36-icons'
 import { useCMA, useSDK } from '@contentful/react-apps-toolkit'
 
+import { EntryListSearch } from '../../../../components/EntryListSearch'
 import { mapLocalesToFieldApis } from '../../utils'
 
-const SEARCH_DEBOUNCE_TIME_IN_MS = 300
 const LIST_ITEM_CONTENT_TYPE_ID = 'genericListItem'
```

Comment:
Maybe not necessary to have that in const, only used once..

Reply:
I removed that line